src/js/edu-benefits/components/SubmitMessage.jsx

```diff
@@ -8,6 +8,13 @@ export default class SubmitMessage extends React.Component {
     this.state = { isExpanded: false };
     this.handleClick = this.handleClick.bind(this);
   }
+  componentDidMount() {
+    const title = document.querySelector('.edu-page-title');
```

Comment:
I don't think it would help much here, since the four places where this is happening are different. It will probably be useful for hca, though, since we'll have to make the same change in a bunch of places.